test/commands/passphrase/encrypt.test.js

```diff
@@ -0,0 +1,190 @@
+						source: undefined,
+						repeatPrompt: true,
+					},
+				});
+				return expect(printMethodStub).to.be.calledWithExactly({
+					encryptedPassphrase: encryptedPassphraseString,
+				});
+			});
+	});
+
+	describe('passphrase:encrypt --outputPublicKey', () => {
+		setupTest()
+			.stdout()
+			.command(['passphrase:encrypt', '--outputPublicKey'])
+			.it('should encrypt passphrase and output public key', () => {
+				expect(
+					cryptography.encryptPassphraseWithPassword,
+				).to.be.calledWithExactly(
+					defaultInputs.passphrase,
+					defaultInputs.password,
+				);
+				expect(
+					cryptography.stringifyEncryptedPassphrase,
+				).to.be.calledWithExactly(encryptedPassphraseObject);
+				expect(getInputsFromSources.default).to.be.calledWithExactly({
+					passphrase: {
+						source: undefined,
+						repeatPrompt: true,
+					},
+					password: {
+						source: undefined,
+						repeatPrompt: true,
+					},
+				});
+				return expect(printMethodStub).to.be.calledWithExactly({
+					encryptedPassphrase: encryptedPassphraseString,
+					...defaultKeys,
+				});
+			});
+	});
+
+	describe('passphrase:encrypt --passphrase=pass:123', () => {
+		setupTest()
+			.stdout()
+			.command(['passphrase:encrypt', '--passphrase=pass:123'])
+			.it('should call print with the user config', () => {
+				expect(
+					cryptography.encryptPassphraseWithPassword,
+				).to.be.calledWithExactly(
+					defaultInputs.passphrase,
+					defaultInputs.password,
+				);
+				expect(
+					cryptography.stringifyEncryptedPassphrase,
+				).to.be.calledWithExactly(encryptedPassphraseObject);
+				expect(getInputsFromSources.default).to.be.calledWithExactly({
+					passphrase: {
+						source: 'pass:123',
+						repeatPrompt: true,
+					},
+					password: {
+						source: undefined,
+						repeatPrompt: true,
+					},
+				});
+				return expect(printMethodStub).to.be.calledWithExactly({
+					encryptedPassphrase: encryptedPassphraseString,
+				});
+			});
+	});
+
+	describe('passphrase:encrypt --passphrase=pass:123 --password=pass:456', () => {
+		setupTest()
+			.stdout()
+			.command([
+				'passphrase:encrypt',
+				'--passphrase=pass:123',
+				'--password=pass:456',
+			])
+			.it(
+				'should encrypt passphrase from passphrase and password flags',
+				() => {
+					expect(
+						cryptography.encryptPassphraseWithPassword,
+					).to.be.calledWithExactly(
+						defaultInputs.passphrase,
+						defaultInputs.password,
+					);
+					expect(
+						cryptography.stringifyEncryptedPassphrase,
+					).to.be.calledWithExactly(encryptedPassphraseObject);
+					expect(getInputsFromSources.default).to.be.calledWithExactly({
+						passphrase: {
+							source: 'pass:123',
+							repeatPrompt: true,
+						},
+						password: {
+							source: 'pass:456',
+							repeatPrompt: true,
+						},
+					});
+					return expect(printMethodStub).to.be.calledWithExactly({
+						encryptedPassphrase: encryptedPassphraseString,
+					});
+				},
+			);
+	});
```

Comment:
What about when both come from stdin?

Reply:
stdin should be covered by two flags case because it's stubbed.